README.md

```diff
@@ -1,7 +1,16 @@
 # TFExplorer
+[![Build Status](https://travis-ci.org/hardlyhuman/TFExplorer.svg?branch=master)](https://travis-ci.org/hardlyhuman/TFExplorer)
+![alt text](https://patrolavia.github.io/telegram-badge/chat.png "Logo title")
+**TFExplorer** enables developers to quickly find out the relevant TensorFlow API's for their needs & requirements.
+It basically deals with different tools for TensorFlow API's .
+This is the repository for backend code.
+The stack is:
```

Comment:
Add version numbers after django and python. You will find django version in the file requirements.txt. Python version is 3.8